cogs/secretSanta.py

```diff
@@ -0,0 +1,145 @@
+import nextcord
+from nextcord.ext import tasks, commands
+import time
+from datetime import *
+import random
+
+msg = ''
+msg_id = ''
+participatingList = []
+b = ''
+santas =[] 
+guild_id = 0
+messagedFlag = False

```

Comment:
Do not create variables outside of the `santa` class. If you add it under `santa`'s `__init__` declaration, you can refer to them in any other declaration within the class.